### PR TITLE
Improve active learning documentation

### PR DESCRIPTION
**Pull request by michal-lightly**
## What changed

* Add descriptions Lightly Worker active learning parameters, add cross references
* Improved formatting
* Stylistic reformulations

## How was it tested
* Generated html doc locally

### CODE REVIEW

**Comment by michal-lightly** [comment]
Before and after examples:

Screenshot 2022-07-05 at 18 18 26

Screenshot 2022-07-05 at 18 18 55

File: docs/source/getting_started/active_learning.rst [hunk 1]
```diff
@@ -157,8 +157,9 @@ Use this list to get predictions on the unlabeled images.
 **Important:** The predictions need to be in the same order as the filenames in the
 list returned by the `ActiveLearningAgent`.
 
-For classification, the predictions need to be in a numpy array and normalized,
-such that the rows sum to one. Then, create a scorer object like so:
+Next, create a scorer object depending on your task (see :ref:`lightly-active-learning-scorers` below).
+For instance, for classification, the predictions need to be in a numpy array and normalized,
+such that the rows sum to one:
```
Comment (michal-lightly) [inline docs/source/getting_started/active_learning.rst:162]: The text here was written just for classification before introducing scorers. Rewrote to be more general.

File: docs/source/getting_started/active_learning.rst [hunk 2]
```diff
@@ -200,40 +201,38 @@ but without performing a selection. This is also easily possible:
...
-The differ in the number of class confidences they take into account.
+and also explained in https://towardsdatascience.com/uncertainty-sampling-cheatsheet-ec57bc067c0b.
+For all of them, the score is the highest if all classes have equal probability,
+and the score is 0.0 if the model assigns 100% probability to a single class.
+They differ in the number of class confidences they take into account.
 
-- **uncertainty_least_confidence**:
-    This score is 1 - the highest confidence prediction. It is high (close to `1.0`)
+- **uncertainty_least_confidence**
+    This score is 1.0 minus the highest confidence prediction. It is high (close to 1.0)
```
Comment (michal-lightly) [inline docs/source/getting_started/active_learning.rst:224]: Avoid confusion between minus and a dash.

File: docs/source/getting_started/active_learning.rst [hunk 3]
```diff
@@ -264,22 +263,24 @@ Currently, the following scorers are available:
...
 
 - **classification_scores**
   These scores are computed for each object detection per image out of
   the class probability prediction for this detection. Then, they are reduced
   to one score per image by taking the maximum. In particular we support:
+
   - **uncertainty_least_confidence**
   - **uncertainty_margin**
   - **uncertainty_entropy**
+
```
Comment (michal-lightly) [inline docs/source/getting_started/active_learning.rst:283]: This was wrongly formatted - empty lines are needed to get a bulleted list.

File: docs/source/docker/advanced/active_learning.rst [hunk 4]
```diff
@@ -33,15 +33,20 @@ In order to do active learning with Lightly, you will need the following things:
...
+- **active_learning.score_name**: Choose from the available scores: :ref:`lightly-active-learning-scorers`
+
+  - :code:`uncertainty_least_confidence`
+  - :code:`uncertainty_margin`
+  - :code:`uncertainty_entropy`
 
 Here's an example of how to configure an active learning run:
 
 .. literalinclude:: code_examples/python_run_active_learning.py
+    :emphasize-lines: 78, 83-86
```
Comment (guarin) [inline docs/source/docker/advanced/active_learning.rst:49]: 🤩

**Review by guarin — APPROVED** [review]
Looks good!

**Pull request merged**